ui/static/app/record/record.ts

```diff
@@ -18,6 +18,7 @@ export class Record {
 
   public static recordTypes: Array<{0: string, 1: string}> = [
     ["A", "IPv4 address: e.g. example.com -> 1.2.3.4"],
+    ["AAAA", "IPv6 address: e.g. example.com -> 2001:0db8:85a3:0000:0000:8a2e:0370:7334"],
```

Comment:
ex. `["A", "IPv4 address: e.g. example.com -> 1.2.3.4", "1.2.3.4"],` - and use the last value as a placeholder in `record-detail.component.ts` instead of `this.inputContent.nativeElement.placeholder = "1.2.3.4";` (ex. `this.inputContent.nativeElement.placeholder = <here get value from the list depending on type>`;